draft-ietf-tls-tls13.md

```diff
@@ -2494,7 +2541,7 @@ for each scenario:
 
 | Mode | Handshake Context | Base Key |
 |------|-------------------|----------|
-| 0-RTT | ClientHello | early_traffic_secret|
+| PSK | ClientHello prefix with 0s for extension body | pre-shared key |
```

Comment:
Nit: Perhaps extension body -> hello_finished body, so it's clearer which extension?